pkg/component/etcd/lease/values_helper.go

```diff
@@ -16,16 +16,17 @@ package lease
 
 import (
 	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// GenerateValues generates `lease.Values` for the lease component with the given parameters.
+// GenerateValues generates `lease.Values` for the lease component
 func GenerateValues(etcd *druidv1alpha1.Etcd) Values {
 	return Values{
 		BackupEnabled:          etcd.Spec.Backup.Store != nil,
 		EtcdName:               etcd.Name,
```

Comment:
I see that in the other components, this is changed to `Name`. Can you do this here too?